src/internal/operations/test/onedrive_test.go

```diff
@@ -822,3 +823,58 @@ func (suite *OneDriveBackupIntgSuite) TestBackup_Run_oneDriveOwnerMigration() {
 		}
 	}
 }
+
+func (suite *OneDriveBackupIntgSuite) TestBackup_Run_oneDriveExtensions() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	var (
+		tenID  = tester.M365TenantID(t)
+		mb     = evmock.NewBus()
+		userID = tester.SecondaryM365UserID(t)
+		osel   = selectors.NewOneDriveBackup([]string{userID})
+		ws     = deeTD.DriveIDFromRepoRef
+		svc    = path.OneDriveService
+		opts   = control.Defaults()
+	)
+
+	opts.ItemExtensionFactory = getTestExtensionFactories()
+
+	osel.Include(selTD.OneDriveBackupFolderScope(osel))
+
+	bo, bod := prepNewTestBackupOp(t, ctx, mb, osel.Selector, opts, version.Backup)
+	defer bod.close(t, ctx)
+
+	runAndCheckBackup(t, ctx, &bo, mb, false)
+
+	bID := bo.Results.BackupID
+
+	deets, expectDeets := deeTD.GetDeetsInBackup(
+		t,
+		ctx,
+		bID,
+		tenID,
+		bod.sel.ID(),
+		svc,
+		ws,
+		bod.kms,
+		bod.sss)
+	deeTD.CheckBackupDetails(
+		t,
+		ctx,
+		bID,
+		ws,
+		bod.kms,
+		bod.sss,
+		expectDeets,
+		false)
+
+	// Check that the extensions are in the backup
+	for _, ent := range deets.Entries {
+		if ent.Folder == nil {
+			verifyExtensionData(t, ent.ItemInfo, path.OneDriveService)
+		}
+	}
```

Comment:
I'm curious if we could build this into the standard `Backup_Run_oneDrive` test (same with sharepoint) instead of running an entirely separate test.  Running another backup entirely for this feels pretty heavy.  Thoughts on folding them together?